src/main/java/io/mosip/digitalcard/controller/DigitalCardController.java

```diff
@@ -0,0 +1,72 @@
+
+import java.io.ByteArrayInputStream;
+
+@RestController
+@Api(tags = "Digital Card")
+public class DigitalCardController {
+
+    @Autowired
+    DigitalCardService digitalCardServiceImpl;
+
+    Logger logger = DigitalCardRepoLogger.getLogger(DigitalCardController.class);
+
+
+    @PostMapping(path = "/idCreateEventHandle/callback/notifyStatus", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiResponses(value = { @ApiResponse(code = 200, message = "Request authenticated successfully") })
+    @PreAuthenticateContentAndVerifyIntent(secret = "${mosip.digitalcard.websub.secret}", callback = "/v1/digitalcard/idCreateEventHandle/callback/notifyStatus", topic = "${mosip.digitalcard.generate.identity.create.websub.topic}")
+    public ResponseEntity<?> identityCreateEvent(@RequestBody EventModel eventModel)  {
+        logger.info("event recieved from websub id: {}, topic : {}",eventModel.getEvent().getId(),eventModel.getTopic());
+        boolean isCreated=digitalCardServiceImpl.createDigitalCard(eventModel);
+        logger.info("successfully created the digitalcard : {}",isCreated);
+        return new ResponseEntity<>("request accepted.", HttpStatus.OK);
+    }
+
+    @PostMapping(path = "/idUpdateEventHandle/callback/notifyStatus", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiResponses(value = { @ApiResponse(code = 200, message = "Request authenticated successfully") })
+    @PreAuthenticateContentAndVerifyIntent(secret = "${mosip.digitalcard.websub.secret}", callback = "/v1/digitalcard/idUpdateEventHandle/callback/notifyStatus", topic = "${mosip.digitalcard.generate.identity.update.websub.topic}")
+    public ResponseEntity<?> identityUpdateEvent(@RequestBody EventModel eventModel)  {
+        logger.info("event recieved from websub id: {}, topic : {}",eventModel.getEvent().getId(),eventModel.getTopic());
+        boolean isCreated=digitalCardServiceImpl.createDigitalCard(eventModel);
+        logger.info("successfully printed the digitalcard : {}",isCreated);
+        return new ResponseEntity<>("request accepted.", HttpStatus.OK);
+    }
+
+    @PostMapping(path = "/credential/callback/notifyStatus", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiResponses(value = { @ApiResponse(code = 200, message = "Request authenticated successfully") })
+    @PreAuthenticateContentAndVerifyIntent(secret = "${mosip.digitalcard.websub.secret}", callback = "/v1/digitalcard/credential/callback/notifyStatus", topic = "${mosip.digitalcard.generate.credential.websub.topic}")
+    public ResponseEntity<?> credentialEvent(@RequestBody EventModel eventModel)  {
+        logger.info("event recieved from websub id: {}, topic : {}",eventModel.getEvent().getId(),eventModel.getTopic());
+        boolean isPrinted=digitalCardServiceImpl.generateDigitalCard(eventModel);
```

Comment:
Lets remove the print reference everywhere

Reply:
taken care